arch/lkl/include/asm/thread_info.h

```diff
@@ -29,6 +38,10 @@ struct thread_info {
 	/* The task for any child that was created during syscall execution.  Only
 	 * valid on return from a clone-family syscall. */
 	struct task_struct *cloned_child;
+	
+ 	/* a linked list of pending signals, pushed onto here as they are detected in move_signals_to_task */
```

Comment:
I think this comment belongs to the field on line 44?

Reply:
Yes, moved